testing/docker-compose.yml

```
@@ -48,7 +48,8 @@ services:
       timeout: 20s
       retries: 3
   s3_proxy:
-    image: ghcr.io/neicnordic/sda-s3proxy:latest
+    image: ghcr.io/neicnordic/sensitive-data-archive:v0.0.130
```

```suggestion
    image: ghcr.io/neicnordic/sensitive-data-archive:${TAG}
```

If you consistently want to test against the latest released version, set the tag as a variable like above.
Then you can start the compose file with the tag set externally:

```cmd
TAG="v0.0.139" docker compose up -d
```

And you can make use of the following command to dynamically get the latest released tag from the `sensitive-data-archive` repo:

```cmd
curl -s https://api.github.com/repos/neicnordic/sensitive-data-archive/tags | jq -r '.[0].name'
```

Putting it all together yields:

```cmd
TAG=$(curl -s https://api.github.com/repos/neicnordic/sensitive-data-archive/tags | jq -r '.[0].name') docker compose up -d
```